internal/app/errors.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"errors"
+
+	v1alpha1 "connectrpc.com/conformance/internal/gen/proto/go/connectrpc/conformance/v1alpha1"
+	"connectrpc.com/connect"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+// ConvertErrorToConnectError converts the given error to a Connect error
+// If err is nil, function will also return nil. If err is not
+// of type connect.Error, a Connect error of code Unknown is returned.
+func ConvertErrorToConnectError(err error) *connect.Error {
```

Comment:
nit: none of these to be exported

Reply:
These are actually in the `app` package since the functionality is needed for both server and client. I can move elsewhere if you like though.

Reply:
Ah, my bad.